src/plugins/guided_onboarding/public/components/guide_panel.test.tsx

```diff
@@ -241,9 +247,9 @@ describe('Guided setup', () => {
 
         await updateComponentWithState(component, mockActiveSearchGuideState, true);
 
-        expect(find('activeStepButton').text()).toEqual('Start');
+        expect(find('onboarding--stepButton--search--step1').text()).toEqual('Start');
```

Comment:
These tests might need to be updated since my search copy PR got merged and the test is using `testGuide` instead of `search`.